samples/azure-quantum/entanglement-swap/README.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Does this replace the existing teleport-chain sample? If possible, I'd recommend combining the two in one folder since they're effectively different representations of the same sample.

Reply:
I think that's a question for @tcNickolas

Reply:
That's really a question for the samples owner :-) I'm not familiar with the limitations of how we publish samples in the gallery and how they have to be structured for that.